packages/react-native-views/src/components/CameraView/hooks/useSuccess.js

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+
+function useSuccess(onSuccess, payload, handleFakeActivity) {
+  const { pictures, camera, sights } = payload;
+  const nbOfSights = sights.length;
+
+  useEffect(() => {
+    const picturesTaken = Object.values(pictures).filter((p) => Boolean(p.source)).length;
```

Comment:
Doesn't `filter` already coerce the result of the function to a boolean?

If we use `nbOfSights` shouldn't we use `nbOfPicturesTaken` here? (I personally prefer `sightsCount` and `takenPicturesCount`)